features/profiles.feature

```diff
@@ -95,3 +98,22 @@ Feature: default command line arguments
   Scenario: specifying a configuration file that doesn't exist
     When I run cucumber-js with `--config doesntexist.js`
     Then it fails
+
+  Scenario: using a JSON file
```

Comment:
No real change needed for this to work, because `require()` works for JSON anyway.